index.js

```diff
@@ -40,16 +42,24 @@ function Arpping({ timeout = 5, includeEndpoints = false, useCache = true, cache
 }
 
 /**
-* Build array of full ip range (xxx.xxx.x.1-255) given example ip address
+* Build array of full ip range (xxx.xxx.xxx.1-255) given example ip address
 * @param {String} ip
+* @param {Number} mask
 */
-Arpping.prototype._getFullRange = function(ip) {
+Arpping.prototype._getFullRange = function(ip, mask = 24) {
     // don't use default assignment so false-y values are overwritten
     ip = ip || this.myIP;
-    var ipStart = ip.substr(0, ip.lastIndexOf('.') + 1);
+    // add mask to ip eg xxx.xxx.xxx.xxx/yy
+    ip = ip + '/' + mask;
+
+    var range = [];
+    var block = new Netmask(ips);
+    range.push(block.base);
+    range.forEach((ip, _, index) => range.push(ip));
```

Comment:
@dkranke Shouldn't this be block.forEach?

Reply:
Your right, i actually only wrote the code and didn't try it.
I'm still in the making of a project that needs it.

Reply:
Thank you for your quick reply. 
I found out these issues while I was testing your PR.  :)

Reply:
I've learned that I should always test before creating a PR